src/groups/mqb/mqbblp/mqbblp_queueconsumptionmonitor.t.cpp

```diff
@@ -63,6 +63,35 @@ static mqbconfm::Domain getDomainConfig()
     return domainCfg;
 }
 
+class QueueConsumptionMonitorTest : public QueueConsumptionMonitor {
+    // This class is a test driver for the 'QueueConsumptionMonitor' class.
+    // It is used to get access to the protected members of the class for
+    // testing purposes.
+  public:
+    // CREATORS
+    QueueConsumptionMonitorTest(QueueState*       queueState,
+                                const LoggingCb&  loggingCb,
+                                bslma::Allocator* allocator)
+    : QueueConsumptionMonitor(queueState, loggingCb, allocator)
+    {
+        // NOTHING
+    }
+
+    ~QueueConsumptionMonitorTest() { reset(); }
+
+    // MODIFIERS
+
+    void alarmEventDispatched()
```

Comment:
All overridden methods of this class should be marked with `BSLS_KEYWORD_OVERRIDE`